### PR TITLE
DOCS-7084-made edits

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

**Review by tbruno — APPROVED** [review]
LGTM 👍

**Review by hannanelson — APPROVED** [review]

File: modules/ROOT/pages/bind-api-configure-service-mesh-CLI.adoc [hunk 1]
```diff
@@ -49,7 +49,7 @@ image::api-id.png[]
 . To retrieve the service name from Kubernetes, with which you want to bind the API, run the command:
 +
 `kubectl get svc -n <namespace>`
-. Verify that your microservice port is named following the requirements requested by Istio: xref:https://istio.io/docs/setup/additional-setup/requirements/[].
+. Verify that your microservice port is named following the requirements requested by xref:http://istio.io/docs/setup/additional-setup/requirements/[Istio].
```
Comment (rddheath) [inline modules/ROOT/pages/bind-api-configure-service-mesh-CLI.adoc]: "following the" -> "according to the"
"requirements requested by" -> "requirements described in"

Comment (rddheath) [inline modules/ROOT/pages/bind-api-configure-service-mesh-CLI.adoc]: Comment on service-mesh-cli-adapter-help.png: By requiring users to view a screenshot to read text, you're breaking accessibility standards, unless you also put that text in alt-text. In this case, that would be impossible. So, what is the purpose of this screenshot?

**Review by rddheath — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/bind-api-configure-service-mesh-CLI.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/bind-api-configure-service-mesh-CLI.adoc]: Done! Thanks!

Comment (priyaayyar) [reply modules/ROOT/pages/bind-api-configure-service-mesh-CLI.adoc]: Took the conversation offline.

**Review by rddheath — APPROVED** [review]

**Pull request merged**